src/main/java/edu/ucsb/cs156/happiercows/entities/Courses.java

```diff
@@ -0,0 +1,21 @@
+package edu.ucsb.cs156.happiercows.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "courses")
+public class Courses {
+    @Id
+    private String courseMajor;
```

Comment:
It's a minor point, but technically the term used at UCSB for this is `subjectArea`.   It's subtly different from `department` or `major`.   

I know it may seem like a small thing, but the best time to fix names is right at the beginning.  Plus both anecdotal experience from software developers as well as some rigorous research shows that:
* Good names help productivity
* Bad names harm productivity.
```suggestion
    private String subjectArea;
```